CONTRIBUTING.md

```diff
@@ -14,70 +14,44 @@ We welcome contributions to our project. Here are a few guidelines that will hel
 
```

Comment:
A few of the links on the page are not working, namely the ones for the contribution page and the links on the table of content.